### ISSUE TITLE
@synch cannot be integrated into a customized ODD with romabeta, and specifications of @synch can also not be edited

### ISSUE THREAD

**Issue by AndreasMunzmay** [TEIC/romajs#132]

**Comment by raffazizzi** [comment]
@AndreasMunzmay can you provide more information or a custom ODD? I was able to do the following in Roma 0.3.1:
* open the page to edit `@sync` on `<when>`
* Change the attribute value to a closed list
* Added one value "example"
* Downloaded valid ODD without error
* Generated correct RelaxNG

**Comment by AndreasMunzmay** [comment]
what I wanted to do is: 
- customize "TEI with Drama"
- using Roma's "Elements" section: activate e.g. in <l> a disabled attribute (like @synch) in its attribute class (here: att.global.linking) 
- try to edit the attribute specs 
-> error message "TypeError: attributeToChange is undefined" appears

Alternatively, I first tried to copy @synch" into the att.global.linking  in Roma's "Attribute Classes" section
At a first glance, everything looks fine, I can edit the attribute specifications.
-> But the <attList> of the exported OOD Costumazition will still keep saying " <attDef ident="synch" usage="opt" mode="delete"/>

**Comment by lb42** [comment]
You cant add @sync to att.global because its alrrady (indirectly) there. You need to include the module in which its defined  though

**Comment by AndreasMunzmay** [comment]
what I wanted to do is to reestablish the @synch in the "TEI with Drama" schema (because in music theatre they often sing different text synchronously). I proceeded as follows:
 
- customize "TEI with Drama"
- using Roma's "Elements" section: activate e.g. in <l> the disabled @synch in its attribute class (here: att.global.linking) 
- try to edit the attribute specs 
-> error message "TypeError: attributeToChange is undefined" appears

Alternatively, I first tried to copy @synch" into the att.global.linking  in Roma's "Attribute Classes" section
At a first glance, everything looks fine, I can edit the attribute specifications.
-> But the <attList> of the exported OOD Costumization will keep saying " <attDef ident="synch" usage="opt" mode="delete"/>

The problem seems to occur not only with synch, but with all attributes deleted individually from classes in the the preset ODDs.
